databricks/koalas/frame.py

```diff
@@ -1110,6 +1110,7 @@ def iterrows(self):
             The index of the row. A tuple for a `MultiIndex`.
         data : pandas.Series
             The data of the row as a Series.
+
```

Comment:
nit: should revert this change?

Reply:
I saw an empty line between `data` and `it` in pandas doc 
https://github.com/pandas-dev/pandas/blob/v0.25.3/pandas/core/frame.py#L860-L910

This function returns a generator `it` and items are `index` and `data`. That might be the reason an empty line here

Reply:
I see, then maybe we should add `Returns` section?
```
        Returns
        -------
        it : generator
            ...
```

Reply:
Fixed!

Reply:
ah, sorry, seems like we can't use both `Yields` and `Returns` at the same time.

```
ValueError: Docstring contains both a Returns and Yields section.
```

Could you revert the last commit?

Reply:
oh interesting. 
sorry I should try to build the doc to test
branch updated